packages/docusaurus-plugin-openapi/src/sidebars/index.ts

```diff
@@ -212,7 +210,7 @@ async function readCategoryMetadataFile(
     const contentString = await fs.readFile(filePath, { encoding: "utf8" });
     const unsafeContent = Yaml.load(contentString);
     try {
-      return validateCategoryMetadataFile(unsafeContent);
+      return unsafeContent as CategoryMetadataFile; // validateCategoryMetadataFile(unsafeContent);
```

Comment:
what should I use here to validate the data?

Reply:
This is fine, we don’t really need to validate. I believe the validate function just gave warnings if your yaml was was missing required fields, it doesn’t do any security checks or anything like that